model-optimizer/extensions/ops/TFResize.py

```diff
@@ -45,10 +43,9 @@ def tf_resize_infer(node: Node):
         len_msg = "Op {} with name {} supports only resize with respect to height and width dimension simultaneously"
         assert len(new_sizes_value) == 2, len_msg.format(node_name, node.op)
 
-        output_shape = int64_array(input_shape.copy())
+        output_shape = input_shape.copy()
 
-        layout = node.graph.graph['layout']
-        output_shape[get_height_dim(layout, input_rank)] = new_sizes_value[0]
-        output_shape[get_width_dim(layout, input_rank)] = new_sizes_value[1]
```

Comment:
Because the TFResize operation has a fixed input tensor layout NHWC (refer to the documentation https://www.tensorflow.org/api_docs/python/tf/image/resize). So in order to calculate output shape we need to use just this layout.